v1/endpoints.md

```diff
@@ -881,6 +882,21 @@ We recommend taking the following precautions when performing free-text searches
 
 Note that the API will always return the latest document version in document searches if you `expanded` the response with `documentVersions` in order to satisfy the most common use case for the search service - free-text searches in the document text.
 
+### Advanced free-text search
+
+By specifying the `enableAdvancedFTS` flag, certain characters will gain special meaning.
+
+- Double quote (") allow you to perform phrase searches for exact matches. For example: "This is my phrase search that requires an exact match"
+- Asterisk (*) and question mark (?) allow you to perform wildcard searches.
+  - Asterisks match zero or more consecutive characters at the beginning, middle, or end of a string. For example: \*master, Documas\*, Doc\*ter
+  - Question marks match a single character at the beginning, middle, or end of a string. For example: ?ocumaster, Documaste?, Do?umaster
+- Tilde (~) allow you to perform fuzzy search and proximity search.
```

Comment:
Here the word _allow_ should be _allows_.

Reply:
Fixed. I've also fixed "Double quoteS" above.